sensors/triggers/params.go

```diff
@@ -183,7 +205,7 @@ func ResolveParamValue(src *v1alpha1.TriggerParameterSource, events map[string]*
 			}
 
 			if err == nil {
-				return &resultValue, nil
+				return &resultValue, errorType, nil
```

Comment:
Is this right? If there's no error, return `errorType`?

The original logic looks very tricky. Can we make it simple like this:

```
if src.Value != nil {
  return *src.Value
} else {
  eventPayload, err = json.Marshal.....
  if err != nil {
    return err
  }
  return string(eventPayload)
}
```

Reply:
have changed the error check just after json.Marshal
about the return, we should return a pointer to the casted eventPayload variable. I think best to write it in memory.

Reply:
and the function returns (*string, string, error)
we can't return error only...